qiskit/test/mock/backends/fake_100q/conf_100q.json

```diff
@@ -0,0 +1 @@
+{"qubit_lo_range":[],"memory":true,"rep_times":[1000],"gates":[],"backend_version":"1.2.4","coupling_map":[[0, 1]],"description":"100 qubit device pulse","max_shots":8192,"n_uchannels":46,"acquisition_latency":[],"max_experiments":900,"conditional_latency":[],"n_registers":1,"dt":2.2222222222222221e-10,"meas_lo_range":[],"basis_gates":["u1","u2","u3","cx","id"],"backend_name":"alt_almaden","u_channel_lo":[],"sample_name":"Qubert","conditional":false,"open_pulse":true,"meas_map":[],"meas_kernels":["boxcar"],"dtm":2.2222222222222221e-10,"discriminators":["linear_discriminator","quadratic_discriminator"],"online_date":"2018-05-10T04:00:00+00:00","simulator":false,"credits_required":true,"n_qubits":20,"local":false,"meas_levels":[1,2],"allow_object_storage":true,"allow_q_circuit":false,"allow_q_object":true,"uchannels_enabled":true,"url":"None","hamiltonian":{"description":"","h_latex":"","h_str":[],"osc":{},"qub":{},"vars":{}},"quantum_volume":8}
```

Comment:
I see these are still just getting started, but would you mind reformatting these 3 files? Not sure what editor you use, but I've used this in the past with Sublime: https://packagecontrol.io/packages/Pretty%20JSON I'm sure there are similar solutions regardless of your env :)